src/server/api/routes/api-router.js

```diff
@@ -4,9 +4,9 @@ const router = express.Router();
 
 // Router imports
 const modulesRouter = require('./modules.router');
-const categoriesRouter = require('./categories.router');
```

Comment:
You should probablly not delete categories router?🤔

Reply:
I should not have done that. Got it🙂